tests/optimization/src/TestInlineConstructors.hx

```diff
@@ -0,0 +1,92 @@
+class InlineClass {
```

Comment:
Missing a test with an assignment to anon structure (provided in PR message).

Reply:
I don't understand what you mean, doesn't `testAnonymousStructureInlining` test for that?

Reply:
No, I mean
```haxe
var a = new Inlinable();
a.job();
var b:{function job():Void;} = a;
b.job();
```

Reply:
What about `testMethodInlining`?

Reply:
Oh, there it is :)